### ISSUE TITLE
In cab controls

### ISSUE THREAD

**Issue by goldenshoes** [modelleicher/FS19_simpleIC#66]
Does anyone else lost all in cab ic controls with new version?

**Comment by modelleicher** [comment]
Is it possible you've got a tow bar / tow weight mod? There's a known konflict between that script and SimpleIC (and some others)

**Comment by Max-La-Menace-FR** [comment]
Hallo !

Haben Sie Zeit gehabt, auf dedizierten Servern nach nicht speichernden Aktionen zu suchen?

Ohne dich natürlich überstürzen zu wollen, fehlt wirklich das kleine Extra ^^

Freundliche Grüße.

**Comment by goldenshoes** [comment]
> 
> 
> Is it possible you've got a tow bar / tow weight mod? There's a known konflict between that script and SimpleIC (and some others)

I don't have that mod and didn't had. So for update of situation: after I remaped IC on/off button on same as it was before I was able X sympols to show up in cab. have you cchanged settings in script that in cab those are default off?

**Comment by JakobMu** [comment]
Hey,
I have the same problem with the new 800 series Vario from cayman718 (LSMC). There is no reaction with activated ic in in-cab mode

**Comment by JakobMu** [comment]
Ich machs mal kurz auf deutsch bevor das noch reines Chaos gibt,
also bei dem 800er is es so das zuerst außen wie innen alles klappt, sobald man aber innen was bedient hat dann außen was bedient und erneut im Traktor argieren will reagiert simple ic nicht, also die kreuze werden rot, beim Klick passiert allerdings nichts, bei Version 9.3.1 und bei der aktuellen so

**Comment by JakobMu** [comment]
Problem gefunden,
Undzwar kollidiert simple IC im incab-Modus mit courseplay. Hier haben wir dann eine Doppelbelgung auf der linken Maustaste durch die beiden Mods (Interaktion courseplay und simple IC). Einfach bei einem der mods die Interaktion von der linken Maustaste auf eine andere legen und schon klappt alles

**Comment by modelleicher** [comment]
> I don't have that mod and didn't had. So for update of situation: after I remaped IC on/off button on same as it was before I was able X sympols to show up in cab. have you cchanged settings in script that in cab those are default off?

Yes since a few versions back the "x" are default off, it was originally intended that way and a few people requestet it so I changed that.
That you remapped the button and then it worked seems like an input conflict

**Comment by modelleicher** [comment]
> Problem gefunden,
> Undzwar kollidiert simple IC im incab-Modus mit courseplay. Hier haben wir dann eine Doppelbelgung auf der linken Maustaste durch die beiden Mods (Interaktion courseplay und simple IC). Einfach bei einem der mods die Interaktion von der linken Maustaste auf eine andere legen und schon klappt alles

Jup diese Doppelbelegungen im LS19 sind schon ewig bekannt und ein immer wiederkehrendes Problem.. :(

**Comment by goldenshoes** [comment]
> 
> 
> > I don't have that mod and didn't had. So for update of situation: after I remaped IC on/off button on same as it was before I was able X sympols to show up in cab. have you cchanged settings in script that in cab those are default off?
> 
> Yes since a few versions back the "x" are default off, it was originally intended that way and a few people requestet it so I changed that.
> That you remapped the button and then it worked seems like an input conflict

well I was happy that it was default on in cab :D

**Comment by modelleicher** [comment]
It should save whatever you set it to per vehicle so if you leave it on on your vehicle it will be always on.
I just noticed that most people play with always off and only activate when they want to use it.

**Comment by goldenshoes** [comment]
> 
> 
> It should save whatever you set it to per vehicle so if you leave it on on your vehicle it will be always on.
> I just noticed that most people play with always off and only activate when they want to use it.
I guess I'm one of few people who prefer it default turned one. but it's not a problem if it saves to be turned on if i leave it like that

**Comment by coolemanUnderscore** [comment]
Hello. Im having the same problem. Suddenly (i guess after i updated Simple-IC) all my in cab triggers are gone. But how do i turn them back on?
